### ISSUE TITLE
SSH connection on iOS fails if the user never trusted the fingerprint

### ISSUE THREAD

**Issue by baltpeter** [tweaselORG/appstraction#53]
If the user never manually connected to the iPhone via SSH and trusted its fingerprint, `sshpass` calls will not run:

```
❯ sshpass -p alpine ssh root@10.0.0.232 uname

```

If you instead use `ssh` directly, you get this:

```
The authenticity of host '[localhost]:22222 ([127.0.0.1]:22222)' can't be established.
ED25519 key fingerprint is SHA256:MgD/jNENZhLQwKPwSzFrzBg/zKBKJE/Pm6D96im7FQg.
Are you sure you want to continue connecting (yes/no/[fingerprint])?
```

**Comment by baltpeter** [comment]
We can work around this with `-o "StrictHostKeyChecking=no"`. While that doesn't sound ideal security-wise, I think it is a fair compromise:

With this option, the host key is [added automatically to `~/.ssh/known_hosts` on the first connection](https://www.shellhacks.com/disable-ssh-host-key-checking/). But it still checks against that. So, if on a later connection, the host key doesn't match the one that was automatically served earlier, it will error.

And let's be honest, here. I certainly didn't verify the iPhones' host keys when I first connected to them, I just entered `yes`. :D
